src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_emitter.hpp

```diff
@@ -5,13 +5,22 @@
 #pragma once
 
 #include "emitters/plugin/x64/jit_emitter.hpp"
+#include "jit_kernel_emitter.hpp"
```

Comment:
If I missed nothing, we don't need this header in the `jit_brgemm_emitter.hpp`. Seems like we have to include only in source file. Otherwise could you please explain the reason of the include?

Reply:
We use `jit_snippets_call_args::amx_tile_config_t` at L20 in this file, so we have to have this include here

Reply:
Oh, I see. For sure! Thanks for the explanation!